addons/binding/org.openhab.binding.russoundserial/src/main/java/org/openhab/binding/russoundserial/handler/RussoundSerialZoneHandler.java

```diff
@@ -0,0 +1,184 @@
+              command);
+        }
+        break;
+      case CHANNEL_ZONEBASS:
+        if (command instanceof IncreaseDecreaseType) {
+          BassLevel level = (command == INCREASE) ? new BassLevel.Up() : new BassLevel.Down();
+          commander().bass(zone, level);
+        } else if (command instanceof DecimalType) {
+          int bassLevel = ((DecimalType) command).intValue();
+          commander().bass(zone, new BassLevel.Set(bassLevel));
+        } else {
+          logger.info("Received a ZONE BASS channel command with a non IncreaseDecreaseType/DecimalType: {}",
+              command);
+        }
+        break;
+      case CHANNEL_ZONETREBLE:
+        if (command instanceof IncreaseDecreaseType) {
+          TrebleLevel level = (command == INCREASE) ? new TrebleLevel.Up() : new TrebleLevel.Down();
+          commander().treble(zone, level);
+        } else if (command instanceof DecimalType) {
+          int level = ((DecimalType) command).intValue();
+          commander().treble(zone, new TrebleLevel.Set(level));
+        } else {
+          logger.info("Received a ZONE TREBLE channel command with a non IncreaseDecreaseType/DecimalType: {}",
+              command);
+        }
+        break;
+      case CHANNEL_ZONEBALANCE:
+        if (command instanceof IncreaseDecreaseType) {
+          Balance level = (command == INCREASE) ? new Balance.Right() : new Balance.Left();
+          commander().balance(zone, level);
+        } else if (command instanceof DecimalType) {
+          int level = ((DecimalType) command).intValue();
+          commander().balance(zone, new Balance.Set(level));
+        } else {
+          logger.info(
+              "Received a ZONE BALANCE channel command with a non IncreaseDecreaseType/DecimalType: {}",
+              command);
+        }
+        break;
+      case CHANNEL_ZONELOUDNESS:
+        if (command instanceof OnOffType) {
+          Loudness level = (command == OnOffType.ON) ? Loudness.ON : Loudness.OFF;
+          commander().loudness(zone, level);
+        } else {
+          logger.info("Received a ZONE LOUDNESS channel command with a non OnOffType: {}",
+              command);
+        }
+        break;
+      case CHANNEL_ZONETURNONVOLUME:
+        if (command instanceof PercentType) {
+          int volumeLevel = ((PercentType) command).intValue();
+          commander().initialVolume(zone, new VolumeChange.Set(volumeLevel));
+        } else {
+          logger.info("Received a ZONE TURNONVOLUME channel command with a non PercentType: {}",
+              command);
+        }
+        break;
+      default:
+        logger.info("Unknown/Unsupported Channel id: {}", channelUID.getId());
+    }
+
+    scheduleRefresh();
+  }
+
+  private void scheduleRefresh() {
+    // Every command requires a status request.
+    if (refreshFuture == null || refreshFuture.isCancelled()) {
+      refreshFuture = scheduler.scheduleAtFixedRate(() -> commander().requestStatus(zone), 0, 100,
```

Comment:
Please use [scheduleWithFixedDelay](http://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ScheduledExecutorService.html#scheduleWithFixedDelay(java.lang.Runnable,%20long,%20long,%20java.util.concurrent.TimeUnit)) which is preferred over scheduleAtFixedRate, see [Coding Guidelines D.3](http://docs.openhab.org/developers/development/guidelines.html#d-runtime-behavior)